model/task_lifecycle.go

```diff
@@ -581,74 +596,57 @@ func MarkEnd(t *task.Task, caller string, finishTime time.Time, detail *apimodel
 
 // UpdateBlockedDependencies traverses the dependency graph and recursively sets each
 // parent dependency as unattainable in depending tasks.
-func UpdateBlockedDependencies(t *task.Task) error {
+// REturns a list of modified task IDs.
+func UpdateBlockedDependencies(t *task.Task) ([]string, error) {
 	const maxStackTraceSize = 5
 	dependentTasks, err := t.FindAllUnmarkedBlockedDependencies()
 	if err != nil {
-		return errors.Wrapf(err, "getting tasks depending on task '%s'", t.Id)
-	}
-
-	if t.IsPartOfSingleHostTaskGroup() && len(dependentTasks) > 0 {
-		depTaskIds := []string{}
-		for _, depTask := range dependentTasks {
-			depTaskIds = append(depTaskIds, depTask.Id)
-		}
-		stack := message.NewStack(2, "").Raw().(message.StackTrace).Frames
-		if len(stack) > maxStackTraceSize {
-			stack = stack[:maxStackTraceSize-1]
-		}
-		grip.Info(message.Fields{
-			"message":                       "blocked task group dependent tasks",
-			"ticket":                        "EVG-12923",
-			"task":                          t.Id,
-			"execution":                     t.Execution,
-			"num_dependent_tasks_to_update": len(dependentTasks),
-			"dependent_task_ids_to_update":  depTaskIds,
-			"stack":                         stack,
-		})
+		return nil, errors.Wrapf(err, "getting tasks depending on task '%s'", t.Id)
+	}
+
+	depTaskIds := []string{}
+	for _, depTask := range dependentTasks {
+		depTaskIds = append(depTaskIds, depTask.Id)
```

Comment:
Could this just be put in the for loop below this? Is the order of task IDs important?